File: ideal-electron-distribution/ideal-electron-distribution.py
def atomic_number(num):
    s=0
    t=[]
    i=1
    while s<num:
        s+=2*i**2;
        if(s<num):
            t.append(2*i**2)
        else:
            t.append(num-sum(t))
        i+=1
    return t

File: ideal-electron-distribution/ideal-electron-distribution.js
function atomicNumber(num){
  var s = 0;
  var i = 1;
  var t = []
  while(s<num){
  s +=(2*Math.pow(i, 2));
  if(s<num){
  t.push(2*Math.pow(i, 2));}else{
  t.push(t!="" ? num - t.reduce((a,b)=>a+b):num )
  }
  i++;
  }
  return t
}